### PR TITLE
Werewolf game add Guard

### PR DESCRIPTION
**Pull request by kevin-meng**
add Guard action and role.

### CODE REVIEW

File: examples/werewolf_game/actions/guard_actions.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+class Protect(Action):
+    """Action: choose a player to protect"""
+
+    PROMPT_TEMPLATE = """
+    It's a werewolf game and you are a guard,
+    you can choose to protect a player, including yourself, then the protected player will not be killed by the Werewolves this night.
+    this is game history:
+    {context}.
+    Attention: you can not protect the same player two nights in a row.  
+    Now, choose one to portect, you will:
```
Comment (garylin2099) [inline examples/werewolf_game/actions/guard_actions.py:12]: typo: portect -> protect

File: examples/werewolf_game/actions/guard_actions.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    """
+
+    def __init__(self, name="Speak", context=None, llm=None):
```
Comment (garylin2099) [inline examples/werewolf_game/actions/guard_actions.py:15]: name="Protect"

**Review by garylin2099 — CHANGES_REQUESTED** [review]

**Comment by kevin-meng** [comment]
add arbiter prompt for evaluation.

File: examples/werewolf_game/start_game.py [hunk 3]
```diff
@@ -1,16 +1,20 @@
 import asyncio
 import platform
 import fire
+import sys
```
Comment (geekan) [inline examples/werewolf_game/start_game.py:4]: Try to avoid such code as much as possible

Comment (kevin-meng) [reply examples/werewolf_game/start_game.py]: ok. got it.  
it's just local testing code. i have delete it.

**Pull request merged**